packages/hakuneko-deb/hakuneko-deb.pacscript

```diff
@@ -0,0 +1,32 @@
+name="hakuneko-deb"
+version="8.3.4"
+nightly="20200705.1"
+arch=("amd64" "arm64" "armhf" "armv7l" "i386")
+
+case "${CARCH}" in
+  amd64)
+    url="https://github.com/manga-download/hakuneko/releases/download/nightly-${nightly}/hakuneko-desktop_${version}_linux_${CARCH}.deb"
```

Comment:
Why is `${nightly}` needed?

Reply:
To have only one update point when the program changes versions. The URL of this repository changes in addition to the value of ${version}, that part which is in ${nightly} every time there is a new release.